packages/local-cli/util/loadMetroConfig.js

```diff
@@ -49,13 +49,12 @@ const getDefaultConfig = (root: string) => {
         'react-native',
         ...plugins.haste.providesModuleNodeModules,
       ],
-      hasteImplModulePath: require.resolve('react-native/jest/hasteImpl'),
     },
     serializer: {
       getModulesRunBeforeMainModule: () => [
         require.resolve('react-native/Libraries/Core/InitializeCore'),
       ],
-      getPolyfills: require('react-native/rn-get-polyfills'),
+      getPolyfills: () => require('react-native/rn-get-polyfills')(),
```

Comment:
why this change? we don't want to pass any args?

Reply:
I've checked Metro configuration and `rn-get-polyfills` and the function does not receive/accept any arguments: https://github.com/facebook/react-native/blob/master/rn-get-polyfills.js#L12